workers/ohsome_quality_analyst/indicators/currentness/metadata.yaml

```diff
@@ -3,16 +3,14 @@ Currentness:
   name: Currentness
   description: |
     Ratio of all contributions that have been edited since 2008 until the current day in relation with years without mapping activities in the same
-    time range.
+    time range. 
     Refers to data quality in respect to currentness.
   label_description:
     red: |
-      More than half of the $elements edited elements were last edited over 4 years ago.
       Be aware that there it is very likely that many map features are outdated. 
       You should carefully check this before using the data as it indicates bad 
       data quality in respect to currentness.
     yellow: |
-      More than half of the edited elements were last edited between 1 and 4 years ago.
       This refers to medium data quality in respect to currentness.
     green: |
       This is a rather high value and indicates that the map features 
```

Comment:
It is likely that most features are up-to-date.

Reply:
Done